frontend/web/components/MyMetadataSelect.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC, useEffect, useState } from 'react'
```

Comment:
Is there a reason why it's called MyMetadata, that would indicate to me it's specific to the user but I think this is for the org?